src/main/java/org/alliancegenome/curation_api/jobs/executors/GeneExpressionExecutor.java

```diff
@@ -39,21 +46,47 @@ public void execLoad(BulkLoadFileHistory bulkLoadFileHistory) {
 
 			bulkLoadFileHistory.setCount(geneExpressionIngestFmsDTO.getData().size());
```

Comment:
I think you need to set up separate counts for annotations and experiments here.  See the GFF Transcript executor for how to do this.  Otherwise your total record count will be less than the completed count as you are counting both annotations and experiments as "Records".

Reply:
Added counts: annotations, experiments